vllm/utils.py

```diff
@@ -748,14 +761,21 @@ def is_pin_memory_available() -> bool:
     return True
 
 
+def _is_cuda_alike_stateless():
+    # Avoid circular import
+    from vllm.platforms import current_platform
```

Comment:
ideally, we should make this line importable from any location

Reply:
Code outside of `vllm.utils` can directly use `current_platform.is_cuda_alike`.